test/test_memmap.py

```diff
@@ -49,15 +49,7 @@ def test_grad():
         torch.bool,
     ],
 )
-@pytest.mark.parametrize(
-    "shape",
-    [
-        [
-            2,
-        ],
-        [1, 2],
-    ],
-)
+@pytest.mark.parametrize("shape", [[2], [1, 2]])
```

Comment:
No changes here, just reformatting some [magic trailing commas](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma)